gremlin-javascript/build/generate.groovy

```diff
@@ -54,15 +54,10 @@ radishGremlinFile = new File("${projectBaseDir}/gremlin-javascript/src/main/java
 // assumes globally unique scenario names for keys with list of Gremlin traversals as they appear
 gremlins = FeatureReader.parseGrouped(Paths.get("${projectBaseDir}", "gremlin-test", "features").toString())
 
-gremlinGroovyScriptEngine = new GremlinGroovyScriptEngine(new GroovyCustomizer() {
-    public CompilationCustomizer create() {
-        return new RepeatASTTransformationCustomizer(new AmbiguousMethodASTTransformation())
-    }
-}, new GroovyCustomizer() {
-    public CompilationCustomizer create() {
-        return new RepeatASTTransformationCustomizer(new VarAsBindingASTTransformation())
-    }
-})
+gremlinGroovyScriptEngine = new GremlinGroovyScriptEngine(
```

Comment:
just to keep these `generate.groovy` files looking as similar as possible, could you also implement this change in for .net and python please?